src/Components/CurrWeatherCard/CurrWeatherCard.jsx

```diff
@@ -13,13 +13,19 @@ export default function CurrWeatherCard({ locationData, currWeather }) {
     let year = selectedDate.toLocaleString('en-GB', {
       year: 'numeric'
     });
+
+    return `${date} ${year}`;
+  }
+
+  function formatTime() {
+    const selectedDate = new Date(currWeather.current.time);
+
     let hour = selectedDate.toLocaleString('en-GB', {
       hour: '2-digit',
       minute: '2-digit',
       hour12: false
     });
-
-    return `${date} ${year} ${hour}`;
+    return `${hour}`;
```

Comment:
`toLocaleString` returns string as default, no need to wrap it in string again

Reply:
done